TodoApp/Tests/TodoApp.Dependency.Tests/Containers/TestContainer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using TodoApp.Contracts.Bootstraps;
+using TodoApp.Contracts.Containers;
+
+namespace TodoApp.Dependency.Tests.Containers
```

Comment:
I would prefer creation of a folder like `Mocks` or `Fakes` so all test implementations of interfaces are under one folder (structure mustn't be flat further on (so `\Mocks\Containers\TestContainer.cs` would be OK)) – I initially thought there are some tests for a container